frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -372,6 +363,17 @@ export const experimentLogic = kea<experimentLogicType>({
                     ).toFixed(1)}%`
                 },
         ],
+        highestProbabilityVariant: [
+            (s) => [s.experimentResults],
+            (experimentResults) => {
+                if (experimentResults) {
+                    const maxValue = Math.max(...Object.values(experimentResults.probability))
+                    return Object.keys(experimentResults.probability).find(
+                        (key) => experimentResults.probability[key] === maxValue
```

Comment:
This can get annoying with floating point equalities

```suggestion
                        (key) => Math.abs(experimentResults.probability[key] - maxValue) < Number.EPSILON
```